modules/scaleform.lua

```diff
@@ -0,0 +1,224 @@
+---@class Scaleform : OxClass
+---@field name string
+---@field scaleform number
+---@field draw boolean
+---@field target number
+---@field targetName string
+---@field handle number
+local Scaleform = lib.class('Scaleform')
+
+---@param details table | string
+---@return nil
+---@description Create a new scaleform class
+function Scaleform:constructor(details)
+    local detailProvided = type(details) == "table"
+    self.name = detailProvided and details.name or details -- Set the name
```

Comment:
`name` is only used to create the `handle`, is there any benefit to not keeping this as just a local variable?